.travis.yml

```diff
@@ -10,10 +10,14 @@ python:
   - "2.7"
 
 # command to install dependencies
-install: "pip install -Ue .[test,build]"
+install: 
+  - "pip install -Ue .[test,build]"
+  - pip install pep8==1.5.7
 
 # command to run tests, e.g. python setup.py test
-script: nosetests --with-cov --cov=testfixtures
+script:
+  - nosetests --with-cov --cov=testfixtures
+  - pep8
```

Comment:
I think you should read that paragraph about the purpose of pep8 ;-)
Don't get me wrong, I'm overjoyed that someone has taken the time to fix all the stuff I wish I could (mainly the spaces after commans, etc), but the tool will require tonnes of config to match the coding style I want to end up with.